template/src/index.js

```diff
@@ -2,6 +2,8 @@ import "babel-polyfill";
 import { registerHtml, start } from "tram-one"
 import ColorHeader from "./components/ColorHeader"
 import "./styles.css"
+import "core-js/stable";
+import "regenerator-runtime/runtime";
```

Comment:
these dependencies need to be added to the package.json in the template.
we should also be removing the first line in this file (`import "babel-polyfill"`), and we should be including these at the top, not as the last imports